src/js/jsx/sections/export/ExportList.jsx

```diff
@@ -141,6 +144,9 @@ define(function (require, exports, module) {
 
             this.getFlux().actions.export.updateExportAsset(
                 this.props.document, this.props.layers, this.props.index, props);
+
+            headlights.logEvent("edit", "export-settings-quality",
+                "Setting: " + props.format + " Quality: " + props.quality);
```

Comment:
@mcilroyc and I talked about this one and it seems unnecessary to log when people change a format or quality, but rather just log what they end up exporting.